src/index.css

```diff
@@ -161,6 +161,20 @@ footer {
   border: 2px solid;
 }
 
+.comment-modal {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 5px;
+  width: 95%;
+  height: auto;
+  padding: 25px 10px;
+  border-radius: 10px;
+  margin: 0 auto;
```

Comment:
We are almost there 👍🏿 The functionality is 💯 Just that because of CSS, the page now appears as shown here 
![kenneth1](https://github.com/ometman/JS-capstone-kanban/assets/129378066/7238b9bd-2817-40c5-bacf-3246cd1832f2)

Reply:
The comment button is not totally visible to the user. I might suspect that maybe you were styling this based on your screen size and you made the height of the comment-modal class to be auto. This should have worked but seems some other property is causing it to behave unexpectedly. Please kindly check again